packages/paste-website/src/pages/foundations/typography/index.mdx

```diff
@@ -0,0 +1,692 @@
+  <Text fontWeight="fontWeightBold" fontSize="fontSize40">
+    Don't
+  </Text>
+</Box>
+<ResponsiveImage
+  src={typography10}
+  alt="don't: wrap headings and paragraphs so that one word is wrapped by itself on a new line, or so that headings stay on one line and look disjointed from the paragraph"
+/>
+
+For this example, you can see the top example has a nice stacked heading that makes the headline and the body copy a compact unit.
+
+In the below examples, you can see one heading and one word in the paragraph are alone, which is harder to scan, but also feels unrefined. The second example has a one line headline which feels disjointed from the rest of the content, as the paragraph line length is shorter. This makes the heading not feel as visually impactful as when it could go to two lines. We want to use design to not only add functionality, but to make a statement.
+
+### Text wrap (line length)
+
+We want to create comfortable reading for our customers. We suggest no more than 120 characters per line. When working with line lengths we want to create short, but efficient lines of text, so the content is easier to scan, it's readable, and not overwhelming.
+
+<Box display="flex" justifyContent="center" marginBottom="space40">
+  <Text fontWeight="fontWeightBold" fontSize="fontSize40">
+    Do
+  </Text>
+</Box>
+<ResponsiveImage src={typography11} alt="do: keep characters per line to a reasonable amount" />
+
+<Box display="flex" justifyContent="center" marginBottom="space40">
+  <Text fontWeight="fontWeightBold" fontSize="fontSize40">
+    Don't
+  </Text>
+</Box>
+<ResponsiveImage src={typography12} alt="don't: let sections of content become to wide and short, or narrow and tall" />
+
+It's also good practice to give more padding (space) if your content is within a container. Just like us humans, our type doesn’t like to feel claustrophobic.
+
+<Box display="flex" justifyContent="space-around" marginBottom="space40">
+  <Text fontWeight="fontWeightBold" fontSize="fontSize40">
+    Do
+  </Text>
+  <Text fontWeight="fontWeightBold" fontSize="fontSize40">
+    Don't
+  </Text>
+</Box>
+
+<ResponsiveImage
+  src={typography13}
+  alt="do: make use of padding in text content areas; don't: let paragrapht text reach the borders of its parent component"
+/>
+
+## Let’s play with type ✨
+
+### 3-minute type zhush
+
+- Stack headlines into two or even three lines instead of keeping them one long line (44 characters or more). This helps create a compact reading unit.
+- A good rule is to use no more than 3 type styles (semibold, medium, regular fonts). This helps blend type styles properly and not get carried away.
+- Be generous with the spacing around your type. It thrives when it has room to breathe. We suggest 8–32 pixels between type if they are friends, of course 🙂. [Check out the spacing and layout guidelines for more info.](/foundations/spacing-and-layout)
+
+### Elevating your designs with type
```

Comment:
thought: it definitely threw me for a second that the after came before the before.

Reply:
@libertycarlin